Sources/Administrator/InputHandler.cpp

```diff
@@ -0,0 +1,121 @@
+//
+// Created by daria on 21.05.16.
+//
+
+#include "InputHandler.h"
+#include <iostream>
+#include <cstring>
+
+void InputHandler::run()
+{
+    std::cout << "Waiting for a command:\n";
+    std::string command;
+    std::string name;
+
+    while ( name != "exit")
+    {
+        std::getline (std::cin,command);
+
+        if ( command.find(" ") == std::string::npos )
+            name = command;
+        else name = command.substr(0,command.find_first_of(" "));
+
+        if ( name == "help") {
+        printHelp();
```

Comment:
Let's keep indentation consistent across the project - 2 spaces, google style code in Clion.